Sources/PackageDescription/Package.swift

```diff
@@ -147,3 +147,15 @@ private func dumpPackageAtExit(_ package: Package, fileNo: Int32) {
     dumpInfo = (package, fileNo)
     atexit(dump)
 }
+
+// MARK: URL Normalization
+
+extension String {
+    private func normalizedUrl() -> String {
```

Comment:
Swift API Design Guideline says it should be a func. at least `lowerCased()` is like this.
But I would say it should be `normalizedURL()` since `URL` is an acronym and acronyms should always be CAPS!